src/libs/store/actions/subscription.ts

```diff
@@ -1,31 +1,37 @@
-import { ContributorId } from 'libs/domain/contributor';
+import { Contributor } from 'libs/domain/contributor';
 import { Level } from 'libs/utils/Logger';
 import { ActionMeta, BaseAction, ErrorAction } from '.';
 
+export type ContributorActionMeta = ActionMeta & { manual?: boolean };
+
 export interface ContributorAction extends BaseAction {
-  payload: ContributorId;
+  payload: Contributor['id'];
+  meta: ContributorActionMeta;
 }
 
 export const SUBSCRIBE = 'SUBSCRIBE';
-export const subscribe = (contributorId: ContributorId, meta?: ActionMeta) => ({
+export const subscribe = (
+  contributorId: Contributor['id'],
+  meta?: ContributorActionMeta
+) => ({
   type: SUBSCRIBE as typeof SUBSCRIBE,
   payload: contributorId,
-  meta
+  meta: { manual: true, ...meta }
```

Comment:
it work even with ` meta: { manual: true, ...meta }` ? `put(subscribe(contributorId, { manual: false }))`

Reply:
Sorry but I don't understand what you mean :sweat_smile:

Reply:
i mean you give to the action `{manual: false}`, but it's already by default on `{manual: true}`, so i ask if its still working ?